db/migrate/20200422114908_add_intent_key_to_payment.rb

```diff
@@ -0,0 +1,5 @@
+class AddIntentKeyToPayment < ActiveRecord::Migration[5.2]
+  def change
+    add_column :spree_payments, :intent_client_key, :string
```

Comment:
```suggestion
    add_column :spree_payments, :intent_client_key, :string, index: true
```

Reply:
On that flow, when we remove intent_client_key after payment processing indexing that field may be overpriced.